### ISSUE TITLE
Create an entry and delete entry in same component

### ISSUE THREAD

**Issue by bfjare** [putyourlightson/craft-sprig#99]
So sorry to bug you, however you do have a new sponsor now!

I know I am probably just missing something stupid, I have an sprig form and a sprig button since using another form to delete just deletes all other entries since all forms are submitted. So I made a sprig button but I cant seem to get it to delete the entry and throws 404 errors in console.log?  Am I missing something stupid?

`<button 
					sprig 
					s-method="post" 
					s-action="entries/delete-entry" 
					s-val:entryId="{{ r.id }}" 
					s-val:sectionId="14"
					class="btn btn-primary delete ml-1"
					s-replace="#client-listed-recommendations"
				>
					<i class="fas fa-trash-alt"></i>
				</button>`

**Comment by bencroker** [comment]
Thank you for becoming a Github sponsor 🎉

Try reading the response in the network tab of the browser console to see what is going wrong. Having `devMode` enabled should give you a detailed error message.

**Comment by bfjare** [comment]
**Error: Request missing required body param**

Which makes me think the params are not correct. I took another look at the button in the inspector and saw this.

<img width="1065" alt="Screen Shot 2021-02-04 at 4 50 11 PM" src="https://user-images.githubusercontent.com/11744681/106969964-28189b80-6709-11eb-90b6-2d6d0a289e0e.png">

I noticed s-val:entryid="2225" and  s-val:sectionid="14" are changed to all lower case I know this breaks in the entry form if I make that all lower case. Is it possible it could be breaking this as well?
